homeassistant/components/tahoma/cover.py

```diff
@@ -28,6 +28,7 @@
 HORIZONTAL_AWNING = "io:HorizontalAwningIOComponent"
 
 TAHOMA_DEVICE_CLASSES = {
+    "io:VerticalInteriorBlindVeluxIOComponent": DEVICE_CLASS_BLIND,
```

Comment:
It looks like this list is alphabetical ordered. Please add this line after `"io:VerticalExteriorAwningIOComponent": DEVICE_CLASS_AWNING,`